src/utils/bulkUpdateUtils.js

```diff
@@ -23,24 +23,23 @@ export function getElevations() {
           const updatedHydrantsMap = Immutable.Map(_.keyBy(updatedHydrants, h => h.get('id')));
           store.dispatch({ type: DATA_IMPORTED, data: { hydrants: updatedHydrantsMap } });
         }
+        return `Fetched Elevation for ${needsElevation.length} hydrants`
```

Comment:
Returning just a number here feels better for me; let the caller use that to determine the message. Also, use `updatedHydrantsMap.size` instead of `needsElevation.length` because some locations might not be able to fetch elevation.